src/components/TreeSpeciesCard.js

```diff
@@ -21,7 +21,7 @@ function TreeSpeciesCard(props) {
       sx={{
         borderRadius: 4,
         overflow: 'hidden',
-        mb: [2, 4],
+        my: [2, 4],
```

Comment:
@RubenSmn let's remove these margin from the component and let the client to deal with the spacing, in this way, we make the component more re-useable.

Reply:
So, remove margin from compnent and put it into the invoker of this component